### PR TITLE
Fix macos supported releases table

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

Fix the table of supported OS versions.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/install/macos.md](https://github.com/dotnet/docs/blob/e79788a0db639555956d6a2c7e957339e3069d0b/docs/core/install/macos.md) | [Install .NET on macOS](https://review.learn.microsoft.com/en-us/dotnet/core/install/macos?branch=pr-en-us-38355) |
| [docs/core/install/remove-runtime-sdk-versions.md](https://github.com/dotnet/docs/blob/e79788a0db639555956d6a2c7e957339e3069d0b/docs/core/install/remove-runtime-sdk-versions.md) | [How to remove the .NET Runtime and SDK](https://review.learn.microsoft.com/en-us/dotnet/core/install/remove-runtime-sdk-versions?branch=pr-en-us-38355) |

### CODE REVIEW

**Review by gewarren — APPROVED** [review]

File: docs/core/install/remove-runtime-sdk-versions.md [hunk 1]
```diff
@@ -109,24 +97,45 @@ sudo rm -rf /usr/share/dotnet/sdk/6.0.406
...
-> [!IMPORTANT]
-> If you're using an Arm-based Mac, such as one with an M1 chip, review the directory paths described in [Install .NET on Arm-based Macs](macos.md#arm-based-macs).
+> The version directories may not match the "version" you're uninstalling. The individual runtimes and SDKs that are installed with a single .NET release may have different versions. For example, you may have installed ASP.NET Core 8 Runtime, which installed the 8.0.2 ASP.NET Core runtime and the 8.0.8 .NET runtime. Each has a different versioned directory. For more information, see [Overview of how .NET is versioned](../versions/index.md).
 
 ::: zone-end
 
-::: zone pivot="os-linux"
+::: zone pivot="os-macos"
+
+When you manually install .NET, it's generally installed to the `/usr/local/share/dotnet/` or the `$HOME/.dotnet` directory. The SDK, runtime, and .NET host, are installed into separate sub directories. These "component" directories contain a directory for each version of .NET. By removing the versioned directories, you remove that version of .NET from your system. These directories may vary depending on your Linux distribution.
```
Comment (gewarren) [inline docs/core/install/remove-runtime-sdk-versions.md:106]:
```suggestion
When you manually install .NET, it's generally installed to the `/usr/local/share/dotnet/` or the `$HOME/.dotnet` directory. The SDK, runtime, and .NET host are installed into separate sub directories. These "component" directories contain a directory for each version of .NET. By removing the versioned directories, you remove that version of .NET from your system. These directories may vary depending on your Linux distribution.
```

File: docs/core/install/remove-runtime-sdk-versions.md [hunk 2]
```diff
@@ -109,24 +97,45 @@ sudo rm -rf /usr/share/dotnet/sdk/6.0.406
...
+There are three commands you can use to discover where .NET is installed: `dotnet --list-sdks` for SDKs, `dotnet --list-runtimes` for runtimes, and `dotnet --info` for everything. These commands don't list the .NET host. To determine which hosts are installed, check the `/usr/local/share/dotnet/host/fxr/` directory. The following list represents the directories of a specific version of .NET, where the `$version` variable represents the version of the .NET:
+
+- **SDK**:
 
-.NET Core installations earlier than 2.0 didn't uninstall the .NET host application when the SDK was uninstalled using the package manager. Using `apt-get`, the command is:
+  `/usr/local/share/dotnet/sdk/$version/`
+
+- **Runtime**:
+
+  The runtime is based on specific .NET product runtimes, such as `Microsoft.AspNetCore.All` or `Microsoft.NETCore.App` (the .NET runtime specifically). These are installed to the `/usr/local/share/dotnet/shared/$product/$version` directory, where `$product` is the product runtime. For example, you may see the following directories:
```
Comment (gewarren) [inline docs/core/install/remove-runtime-sdk-versions.md:116]:
```suggestion
  The runtime is based on specific .NET product runtimes, such as `Microsoft.AspNetCore.All` or `Microsoft.NETCore.App` (the .NET runtime specifically). These are installed to the `/usr/local/share/dotnet/shared/$product/$version` directory, where `$product` is the product runtime. For example, you might see the following directories:
```

File: docs/core/install/remove-runtime-sdk-versions.md [hunk 3]
```diff
@@ -109,24 +97,45 @@ sudo rm -rf /usr/share/dotnet/sdk/6.0.406
...
+Use the `rm -rf` command to remove a version of .NET. For example, to remove the 6.0.406 SDK, run the following command:
 
 ```bash
-apt-get remove dotnet-host
+sudo rm -rf /usr/local/share/dotnet/sdk/6.0.406
 ```
 
-There's no version attached to `dotnet-host`.
+> [!IMPORTANT]
+> The version directories may not match the "version" you're uninstalling. The individual runtimes and SDKs that are installed with a single .NET release may have different versions. For example, you may have installed ASP.NET Core 8 Runtime, which installed the 8.0.2 ASP.NET Core runtime and the 8.0.8 .NET runtime. Each has a different versioned directory. For more information, see [Overview of how .NET is versioned](../versions/index.md).
```
Comment (gewarren) [inline docs/core/install/remove-runtime-sdk-versions.md:135]:
```suggestion
> The version directories might not match the "version" you're uninstalling. The individual runtimes and SDKs that are installed with a single .NET release might have different versions. For example, you might have installed ASP.NET Core 8 Runtime, which installed the 8.0.2 ASP.NET Core runtime and the 8.0.8 .NET runtime. Each has a different versioned directory. For more information, see [Overview of how .NET is versioned](../versions/index.md).
```

File: docs/core/install/remove-runtime-sdk-versions.md [hunk 4]
```diff
@@ -109,24 +97,45 @@ sudo rm -rf /usr/share/dotnet/sdk/6.0.406
 ```
 
 > [!IMPORTANT]
-> The version directories may not match the "version" you're uninstalling. The individual runtimes and SDKs that are installed with a single .NET release may have different versions. For example, you may have installed ASP.NET Core 5 Runtime, which installed the 5.0.2 ASP.NET Core runtime and the 5.0.8 .NET runtime. Each has a different versioned directory. For more information, see [Overview of how .NET is versioned](../versions/index.md).
-
-> [!IMPORTANT]
-> If you're using an Arm-based Mac, such as one with an M1 chip, review the directory paths described in [Install .NET on Arm-based Macs](macos.md#arm-based-macs).
+> The version directories may not match the "version" you're uninstalling. The individual runtimes and SDKs that are installed with a single .NET release may have different versions. For example, you may have installed ASP.NET Core 8 Runtime, which installed the 8.0.2 ASP.NET Core runtime and the 8.0.8 .NET runtime. Each has a different versioned directory. For more information, see [Overview of how .NET is versioned](../versions/index.md).
```
Comment (gewarren) [inline docs/core/install/remove-runtime-sdk-versions.md:100]:
```suggestion
> The version directories might not match the "version" you're uninstalling. The individual runtimes and SDKs that are installed with a single .NET release might have different versions. For example, you might have installed ASP.NET Core 8 Runtime, which installed the 8.0.2 ASP.NET Core runtime and the 8.0.8 .NET runtime. Each has a different versioned directory. For more information, see [Overview of how .NET is versioned](../versions/index.md).
```

**Comment by adegeo** [comment]
Fixes #38355

**Pull request merged**